api/src/main/java/org/openmrs/module/mulagoemr/MulagoEMRActivator.java

```diff
@@ -0,0 +1,46 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.mulagoemr;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.BaseModuleActivator;
+import org.openmrs.module.dataexchange.DataImporter;
+
+/**
+ * This class contains the logic that is run every time this module is either started or shutdown
+ */
+public class MulagoEMRActivator extends BaseModuleActivator {
+	
+	private Log log = LogFactory.getLog(this.getClass());
+	
+	/**
+	 * @see #started()
+	 */
+	public void started() {
+		
+		// install concepts
+		DataImporter dataImporter = Context.getRegisteredComponent("dataImporter", DataImporter.class);
+		
+		dataImporter.importData("MulagoEMRConcepts.xml");
```

Comment:
Change this to the Correct Path. This is pointing to a different location.
https://pastebin.com/zyyC3N4y

Reply:
@slubwama I don't know how I missed that, thanks for catching this. I have also updated the pom files to align to aijar so that it is consistent for later use.